README.md

```diff
@@ -4,7 +4,7 @@ Library that helps you to check if value equals 11.
 
 # Installation
 
-Hope you how to install packages.
+Hope you know how to install packages.
```

Comment:
I think it might be "guess" instead of "know"

Reply:
No, I guess it is exactly "know". @Snegirekk

Reply:
Not sure..

Reply:
I am sure